requirements/2.7/requirements.txt

```diff
@@ -40,9 +40,6 @@ django-tagging==0.3.1
 pytz==2013.8
 
 # Cthulhu
-M2crypto==0.22.3
```

Comment:
Hmm.  So these requirements are removed, presumably because install_salt gives them to you anyway, but install_salt is only called when using vps_bootstrap (i.e. not by normal human developers, presumably)?